src/Imdb/Title.php

```diff
@@ -3008,5 +3008,36 @@ public function real_id()
             return $matches[1];
         }
     }
+    
+    /*
+    * Get the Stars members for this title
+    * @return empty array OR array Stars (array[0..n] of array[imdb,name])
+     * e.g.
+     * <pre>
+     * array (
+     *  'imdb' => '0000134',
+     *  'name' => 'Robert De Niro', // Actor's name on imdb
+     * )
+     * </pre>
+    */
+    public function actor_stars(){
```

Comment:
You can move this function up to (or above) the cast() function if you want. This function's likely more useful for most people.